### PR TITLE
feat: Post Page Design and Make Cover pic and profile pic edit icon w…

### PR DESCRIPTION
**Pull request by yeaseen22**
…ith FrontEnd.

### CODE REVIEW

File: client/src/components/Post/PostUI.js
Comment (AsadAnik) [inline client/src/components/Post/PostUI.js:1]: Why do we need to add all the previously written codes with comments? Doesn't make sense. Please keep it Simple and replace the current one if you are adding a new feature. There is no need to comment out every code previously.
@yeaseen22

File: client/src/components/Profile/ProfileOthers.js
Comment (AsadAnik) [inline client/src/components/Profile/ProfileOthers.js:1]: @yeaseen22 Can we make it more simple? If possible make it. We need to use the best practice here. If you have any plan on mind share with me, please. Or we can having a meeting for this one.

File: client/src/components/commons/Birthdate.js
Comment (AsadAnik) [inline client/src/components/commons/Birthdate.js:1]: @yeaseen22 Make util functions for such like external libraries Moment or e.t.c. The util first approach will help you to make utility become loosely coupled functions or methods or services which can work independently. And if we have to change in the future we can immediately do this without any headache.

File: server/config/multer.js
Comment (AsadAnik) [inline server/config/multer.js:1]: @yeaseen22 Multer is not for Cloudinary. If you want to use Cloudinary services then use a shared or make a library over Cloudinary then in future, we can change it in minutes.

**Review by AsadAnik — COMMENTED** [review]
Please Check the Comments on the codes @yeaseen22

File: client/src/components/commons/Birthdate.js
Comment (yeaseen22) [reply client/src/components/commons/Birthdate.js:1]: I have written  the moment library code correctly and made a PR

File: server/config/multer.js
Comment (yeaseen22) [reply server/config/multer.js:1]: Acknowledge

File: client/src/components/Profile/ProfileOthers.js
Comment (yeaseen22) [reply client/src/components/Profile/ProfileOthers.js:1]: ok meet the requirements and you confirm it

File: server/config/multer.js
Comment (yeaseen22) [reply server/config/multer.js:1]: resolve ti

File: client/src/components/commons/Birthdate.js
Comment (AsadAnik) [reply client/src/components/commons/Birthdate.js:1]: @yeaseen22 That awesome bro. Thanks

File: server/config/multer.js
Comment (AsadAnik) [reply server/config/multer.js:1]: @yeaseen22 Nice. Thanks for improvement here. As we are discussed about you immediately resolved the issue.

**Pull request merged**